subt/artf_node.py

```diff
@@ -6,12 +6,16 @@
 
 import cv2
 import numpy as np
-try:
-    import torch
-    import subt.artf_model
-    from subt.artf_detector import Detector
-except ImportError:
-    print('\nWarning: missing torch!\n')
+TF_DETECTOR = True
```

Comment:
I do not like this switch - ideally I could pass it from JSON configuration and in extreme to run both detector in parallel (if GPU allows that). @jisa what do you think?

Reply:
Ok I will check the json configuration, but the extreme variant could be the best.